allennlp/modules/seq2seq_decoders/seq2seq2_decoder.py

```diff
@@ -0,0 +1,24 @@
+import torch
+
+from allennlp.common import Registrable
+
+
+class Seq2SeqDecoder(torch.nn.Module, Registrable):
+    # pylint: disable=abstract-method
+    """
+    A ``Seq2SeqDecoder`` is a ``Module`` that takes as input a result of ``Seq2SeqEncoder`` and returns a
+    new sequence of vectors.
+
+    Decoder should implement function to perform a single decoding step.
+    """
+
+    def __init__(self, output_dim):
+        super(Seq2SeqDecoder, self).__init__()
+        self._output_dim = output_dim
+
+    def get_output_dim(self) -> int:
```

Comment:
Related to my other comment, we probably don't need this

Reply:
Same here